package.json

```diff
@@ -85,7 +85,7 @@
     "webpack-stream": "^4.0.0"
   },
   "resolutions": {
-    "gulp/**/natives": "1.1.3"
+    "gulp/**/natives": "1.1.6"
```

Comment:
@chaitanya-deep - can we split this part into a separate PR? Just so we don't conflate two different changes `componentWillReceiveProps` & dependency updates. 

Also for the dependency update, I think updating the `yarn.lock` file is a good idea.